content/docs/csidriver/features/powerstore.md

```diff
@@ -760,7 +760,7 @@ If such a node is not available, the pods stay in Pending state. This means they
 Without storage capacity tracking, pods get scheduled on a node satisfying the topology constraints. If the required capacity is not available, volume attachment to the pods fails, and pods remain in ContainerCreating state. Storage capacity tracking eliminates unnecessary scheduling of pods when there is insufficient capacity.
 
 The attribute `storageCapacity.enabled` in `my-powerstore-settings.yaml` can be used to enabled/disabled the feature during driver installation .
-To configure how often driver checks for changed capacity set `storageCapacity.pollInterval` attribute. In case of driver installed via operator, this interval can be configured in the sample files provided [here](https://github.com/dell/dell-csi-operator/tree/master/samples) by editing the `capacity-poll-interval` argument present in the `provisioner` sidecar.
+To configure how often driver checks for changed capacity set `storageCapacity.pollInterval` attribute. In case of driver installed via operator, this interval can be configured in the sample files provided [here](https://github.com/dell/csm-operator/tree/main/samples) by editing the `capacity-poll-interval` argument present in the `provisioner` sidecar.
 
 **Note:**
 >This feature requires kubernetes v1.24 and above and will be automatically disabled in lower version of kubernetes.
```

Comment:
Not necessarily related to this PR but this note can be removed.  For CSM 1.9 we only support 1.26, 1.27, and 1.28